### ISSUE TITLE
Filter by brand in products section

### ISSUE THREAD

**Issue by alfonsolr09** [PrestaShop/PrestaShop#22993]
#### Describe the solution you'd like

I think it would be good that just as we can filter by categories, in the future we can filter by brands or suppliers. since when changing prices of a supplier we can only see the products of that supplier

#### Additional context

Image 

![image](https://user-images.githubusercontent.com/23308427/105818739-91850580-5fb7-11eb-9cb1-681f4d9f3d45.png)

https://prnt.sc/xmg9u9

**Comment by hibatallahAouadni** [comment]
Hello @alfonsolr09 

Thanks for your suggestion! 
Thanks for the suggestion, the Product Team will take a look.
Please stay tuned !

Ping @PrestaShop/product-team , I think it's a good idea to add a filter by brands or suppliers

**Comment by ruchus** [comment]
Hello @alfonsolr09 and @hibatallahAouadni,

I think is useful, but, seems better idea add a new column on grid, like this.

![imagen](https://user-images.githubusercontent.com/6729380/105828855-ed558b80-5fc3-11eb-888d-12862c56d4d5.png)

**Comment by MatShir** [comment]
@prestascott the customization of the tables might be more and more needed 🤔 
Otherwise, it is an interesting feature.

**Comment by alfonsolr09** [comment]
> Hello @alfonsolr09 and @hibatallahAouadni,
> 
> I think is useful, but, seems better idea add a new column on grid, like this.
> 
> ![imagen](https://user-images.githubusercontent.com/6729380/105828855-ed558b80-5fc3-11eb-888d-12862c56d4d5.png)
> 

Yes, i think the same

**Comment by Hlavtox** [comment]
@alfonsolr09 Good idea indeed, but you need a big monitor. 😄 I also added MPN to this list.

![Výstřižek](https://user-images.githubusercontent.com/6097524/105852672-0cb0e080-5fe5-11eb-8e9a-c32c7d5ae5d0.JPG)

**Comment by alfonsolr09** [comment]
Thats no problem, now all people have 21:9 monitor ultrawide 😆
In 2 years everyone will have it
@Hlavtox what is MPN??

**Comment by alfonsolr09** [comment]
> 
> ![Výstřižek](https://user-images.githubusercontent.com/6097524/105852672-0cb0e080-5fe5-11eb-8e9a-c32c7d5ae5d0.JPG)
> 

Thats a test ?? Or have you changed the code??

**Comment by Hlavtox** [comment]
@alfonsolr09 I modified it locally. MPN = manufacturer's part number, people use it to google the product etc. PS team added it officially since 1.7.7.

**Comment by alfonsolr09** [comment]
Do you know the spanish translation on MPN??
if you have the modification done locally you can create a pull request to implement it :) 
@Hlavtox 
Or send me please the code to implement to my shop 🙃🙈

**Comment by Hlavtox** [comment]
@alfonsolr09 Text me on PS slack ;-)

**Comment by alfonsolr09** [comment]
> @ alfonsolr09 Envíame un mensaje de texto en PS slack ;-)

I dont know what is that jeje

**Comment by hibatallahAouadni** [comment]
Hello @alfonsolr09 

Here's the link of [public Slack channel](https://github.com/PrestaShop/open-source/tree/master/slack).
Thanks @Hlavtox for helping other contributors :superhero: 

Thanks!

**Comment by prestascott** [comment]
Hello everyone,

Thanks @alfonsolr09 for your request!

There is a lot of demands about this custom tables/lists features. We will study this deeply with the product team. Also, there is an Epic where you can find all the tables improvement >>> https://github.com/PrestaShop/PrestaShop/issues/20763. 

We're trying to figure out a solution where there's not so much breaking changes for customizing tables/lists in the BO.

Can you please submit your ideas and feedbacks about this feature in our Product Board Portal >>> [here](https://portal.productboard.com/prestashop/1-prestashop-feedback-the-place-to-share-your-feedback-on-prestashop-s-next-features). 
The card is named _"Custom Tables/Lists"_.

Thank you very much! :)
